app/components/Header/DropdownContents/ProfileDropdown/ProfileDropdown.css

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css2?family=Lora&display=swap');
+
+.profileDropdownEl {
+  width: 17rem;
+}
+
+.username {
+  display: flex;
+  margin-bottom: 0.4em;
+  padding: 0.3em 0.6em 0.3em 0.6em;
+  width: fit-content;
+  color: var(--lego-font-color);
+  font-family: 'Lora', serif;
+  font-size: 1.5em;
+  font-weight: bold;
+}
+
+.username:hover {
+  text-decoration: underline;
+}
+
+.profileDropdownEl .dropdownLink {
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+  padding: 0.3em 0.6em 0.3em 0.6em;
+  width: fit-content;
+  border-radius: 0.5em;
+  color: var(--lego-red);
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.1em;
+  transition: background-color 100ms ease-in-out;
+}
+
+.profileDropdownEl .dropdownLink:hover,
+.profileDropdownEl .activeDropdownLink {
+  background-color: rgba(214, 212, 212, 0.3);
```

Comment:
Use variables.

Reply:
I've tried to use variables as much as I can, but found some of them a bit lacking. I'm therefore planning on generally improving `variables.css`. The "red" colors there are incorrect (according to (abakus.no/brand)[https://abakus.no/brand], and we also don't need three variables for the same red color. Perhaps I should even ask the PR committee for some consultation regarding what colors are "correct" to use 🤔